New Term - polypeptide cleavage site

jmedinaebi opened the pull request:
New term for issue https://github.com/HUPO-PSI/psi-mi-CV/issues/459
New terms
* MI:2434 - molecule cleavage site
* MI:2435 - polypeptide cleavage site

Term MI:2434 is child of biological feature, and term MI:2435 is child of term MI:2434.
Both terms with a definition and reference PMID:14755292
Both terms in subset 'PSI-MI_slim'.

sandraorchard commented:
New term is a child of biological feature, as mentioned in the issue. Issue also mentions that 'molecule cleavage site' could be added as a parent, but that term does not exist. Should we create it?

Yes, we may as well. Definition - The location on a molecule where the molecule is  broken down by enzymatic or chemical activity. PMID:14755292
New term has definition in the issue with the PMID mentioned there too, but no aliases or synonym as none were provided. Should we add any? 
I can't think of any -t should be self-explanatory.

New term is in subset 'PSI-MI_slim', as most of MI terms are. Should it be? 

I think (but may be wrong) was how we tagged for import into IntAct so yes.

jmedinaebi commented:
> New term is a child of biological feature, as mentioned in the issue. Issue also mentions that 'molecule cleavage site' could be added as a parent, but that term does not exist. Should we create it?
> 
> Yes, we may as well. Definition - The location on a molecule where the molecule is broken down by enzymatic or chemical activity. PMID:14755292 New term has definition in the issue with the PMID mentioned there too, but no aliases or synonym as none were provided. Should we add any? I can't think of any -t should be self-explanatory.

To clarify, the term 'molecule cleavage site’, is it a child term of biological feature? If so, it’s then the term 'polypeptide cleavage site’, should it be a child term of 'molecule cleavage site’?

So, should the ontology be as follows?
- feature type
  - biological feature
    - molecule cleavage site
      - polypeptide cleavage site

sandraorchard commented:
Yes, your hierarchy is correct

EliotRagueneau commented:
@jmedinaebi please update last-id.txt with the current number, instead of deleting the file

jmedinaebi commented:
> @jmedinaebi please update last-id.txt with the current number, instead of deleting the file

The file didn't exist before. It got created by Protege and I added it by accident, then I deleted. The file was not in the master or develop branch before. I can add it if you prefer we have it though.

jmedinaebi commented:
@EliotRagueneau I just pushed here another commit to fix an issue with the humap search url. The actual url was added as a comment not a label and we could not read the obo file properly, which we do when importing xmls.

Is this ready to be merged?

EliotRagueneau commented:
No `last-id.txt` was added by me on https://github.com/HUPO-PSI/psi-mi-CV/commit/fa1f50f8bf1f5cc3524872842bf92f277ab4565b, it isn't a sideproduct of protege, but something we should keep up to date to be sure we are not declaring twice the same id or skipping one, and that is pretty important specially when 2 people or more are contributing to the files, as the protege sessions do not get shared within the obo file.

So please add back the file and update it to the latest version

EliotRagueneau commented:
From what I can see it looks good to me now. I'm not an expert though ^^' 
I think we can just merge and see if anything breaks

sandraorchard commented:
Assuming nothing breaks - can we have a CV update into the editor. I'm not sure if the CVObject Synch button still works, and I don't want to break anything by a blind rial.
